README.md

```diff
@@ -6,6 +6,22 @@ for legislation related to human rights issues.
 This is being done in conjunction with a non-profit that publishes 
 open comparative datasets on legislation around the world. 
 
+
+## Installation
+`legiscrapor` can be installed with
+
+```sh
+pip install .
+```
+(PyPi installation available soon)
```

Comment:
perhaps presumptuous but if you put all this work in, only a little bit more to get `pip install legislator` + globally available command line functionality like (e.g. `legiscrape ...` or whatever)

Reply:
oh that's a cool idea...I definitely want to think about this more, but at the very least get this initial packaging tried out first!